src/Symfony/Component/Serializer/Mapping/ClassMetadata.php

```diff
@@ -28,7 +28,7 @@ class ClassMetadata implements ClassMetadataInterface
     public $name;
 
     /**
-     * @var AttributeMetadataInterface[]
+     * @var array
```

Comment:
Not really a bug (it just improves static analysis) but it makes sense to apply this on every versions.